apps/docs/components/assistant-ui/thread.tsx

```diff
@@ -43,17 +43,19 @@ export const Thread: FC = () => {
         turnAnchor="top"
         className="aui-thread-viewport relative flex flex-1 flex-col overflow-x-auto overflow-y-scroll scroll-smooth px-4 pt-4"
       >
-        <AssistantIf condition={({ thread }) => thread.isEmpty}>
-          <ThreadWelcome />
-        </AssistantIf>
+        <div className="flex min-h-0 flex-1 flex-col">
```

Comment:
we try to keep the thread.tsx file as minimal as possible. the composer issue was fixed by applying mt-auto style to the footer, much simpler than an extra div